File: fuentes/serverTickets/models/Ticket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ServidorApp.models
{
    public class Ticket
    {
        // Elimina la propiedad FECHA si no se va a utilizar desde el JSON.
        public int ID { get; set; } // Considera si realmente necesitas el ID aquí.

        public string ESTADO { get; set; }
        public string USUARIO_SOLICITANTE { get; set; }
        public string TIPO_AVISO { get; set; }
        public string ORIGEN_AVISO { get; set; }

        public bool URGENTE { get; set; }
        
        public string OBSERVACIONES { get; set; }
        public string DETALLES { get; set; }
    }


}

File: fuentes/android/app/src/main/java/com/tarea3/myapplication/tickets/ui/clases/Ticket.kt
package com.tarea3.myapplication.tickets.ui.clases

import java.time.LocalDate
import java.util.Date

data class Ticket(
    val ID: Int?,
    val ESTADO: String,
    val FECHA: LocalDate?,
    val USUARIO_SOLICITANTE: String,
    val TIPO_AVISO: String,
    val ORIGEN_AVISO: String,
    val URGENTE: Boolean,
    val DETALLES: String,
    val OBSERVACIONES: String
)
data class TicketConImagen(
    val ID: Int,
    val NUMERO_LINEA: Int,
    val DETALLES: String,
    val ESTADO: String?,
    val IMAGEN: String?
)
